src/Entity/Task.php

```diff
@@ -1401,4 +1401,13 @@ public function setDistanceFromPrevious($distanceFromPrevious)
 
         return $this;
     }
+
+    public function getStore(): Store|null
+    {
+        if(!is_null($this->getDelivery())) {
+            return $this->getDelivery()->getStore();
+        }
+
+        return null;
+    }
```

Comment:
is not it the same as `$this->getDelivery()?->getStore()`

Reply:
ah yes indeed i didnt know this PHP syntax

Reply:
I will change